tests/Common/LoggingTest.php

```diff
@@ -55,4 +59,48 @@ public function testAwsLogger(): void
         $options = new \Keboola\StorageApi\Options\FileUploadOptions();
         $client->uploadFile(__DIR__ . '/../_data/files.upload.txt', $options);
     }
+
+    public function testRequestLog(): void
+    {
+        $logsHandler = new TestHandler();
+        $logger = new Logger('tests', [$logsHandler]);
+
+        $client = new Client([
+            'token' => STORAGE_API_TOKEN,
+            'url' => STORAGE_API_URL,
+            'userAgent' => 'Client Testing',
+            'logger' => $logger,
+            'backoffMaxTries' => 1,
+        ]);
+
+        $client->indexAction();
+
+        self::assertTrue($logsHandler->hasDebug(
+            'GET https://connection.keboola.com/v2/storage/ 200'
+        ));
+    }
+
+    public function testFailedRequestLog(): void
+    {
+        $logsHandler = new TestHandler();
+        $logger = new Logger('tests', [$logsHandler]);
+
+        $client = new Client([
+            'token' => STORAGE_API_TOKEN,
+            'url' => STORAGE_API_URL,
+            'userAgent' => 'Client Testing',
+            'logger' => $logger,
+            'backoffMaxTries' => 1,
+        ]);
+
+        try {
+            $client->apiGet('invalid/url');
```

Comment:
self fail? ja vim ze to neni uplne predmetem tohohle testu, ale prijde mi to nepatricny

Reply:
ok, why not